charts/cp-schema-registry/templates/deployment.yaml

```diff
@@ -156,12 +156,12 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             - labelSelector:
                 matchLabels:
-                  app: {{ .Release.Name }}
+                  app: {{ template "cp-schema-registry.name" . }}
```

Comment:
This will generates the `cp-schema-registry` value => a fixed value, regardless of the release !
So, if we have 2 schema-registry inside the same ns, they will overlap ...

NB: same remark for pdb

Reply:
You replaced by the `release` label.
So, it should be safe now.

To be safer and more coherent with the rest of the chart, we should put exactly the same labels than in the `selector` of the Deployment (it's really the same logic).
So: both `app` *and* `release` label.  For the 2 affinity + the pdb.